variables.tf

```
@@ -1,4 +1,26 @@
-variable "name" {
+variable "upstream_registry" {
```

```suggestion
variable "upstream_registry_name" {
```

I would prefer to be more explicit, what do you think? If this is ok, you should update all references.